### DISCUSSION TITLE
Set windows to default ImageGlass

### DISCUSSION THREAD

**Discussion by lpx01** [d2phap/ImageGlass#1129] [General] [answered]
I wonder if you guys have any script so we changed the viewed from standard windows photos to the imageglass. We try to perform this process via GPO and some scripts but when opening the images is still asked to choose which viewer I want to use. Wouldn't you like this message to appear and open directly with imageglass, would you have some script or documentation to support?

**Comment by thenewguy** [comment]
Did you solve this? Can you share the solution to set this non-interactively for all users?

**Comment by d2phap** [comment, marked as answer]
This answer from Allan Holdt: https://groups.google.com/g/imageglass/c/-bkIUt_oJZo/m/D1b8ZGd3AAAJ

> This can be set using the following GPO setting.
>
>The XML file needed can be extracted from a system with the right file associations already set using this command: dism /online /Export-DefaultAppAssociations:"C:\TEMP\DefaultFileAssociations.xml"
> The XML file will include ALL file associations, so edit as needed and place on a fileshare reachable from all clients.

![image](https://user-images.githubusercontent.com/3154213/187190849-09322103-4e88-457d-8490-9be77c3e0643.png)
